harness/determined/estimator/_callback.py

```diff
@@ -29,6 +31,16 @@ def on_checkpoint_end(self, checkpoint_dir: str) -> None:
         """
         pass
 
+    def on_validation_end(self, validation_metrics: Dict[str, Any]) -> None:
+        """
+        Run after every validation..
+
+        .. warning::
+            If distributed or parallel training is enabled, this callback is executed
+            only on the chief GPU (rank = 0).
```

Comment:
Why do we impose this limitation? Wouldn't this be a natural place for e.g. adjusting the optimizer learning rate?